quarkus/tests/integration/src/test/java/org/keycloak/it/storage/database/BasicDatabaseTest.java

```diff
@@ -43,12 +47,31 @@ void testWrongUsername(LaunchResult result) {
     protected abstract void assertWrongUsername(CLIResult cliResult);
 
     @Test
-    @Launch({ "start-dev", "--db-password=wrong" })
+    @Launch({ "start", "--http-enabled=true", "--hostname-strict=false", "--db-password=wrong" })
     void testWrongPassword(LaunchResult result) {
         CLIResult cliResult = (CLIResult) result;
         cliResult.assertMessage("ERROR: Failed to obtain JDBC connection");
         assertWrongPassword(cliResult);
     }
 
     protected abstract void assertWrongPassword(CLIResult cliResult);
+
+    @Order(1)
```

Comment:
I would prefer to make those tests order-agnostic committing to the repo a minimal export. wdyt?

Reply:
Could you elaborate, please? This is the same approach from @DGuhr PR that you approved.